main.go

```diff
@@ -29,60 +22,81 @@ func dbConn() (db *sql.DB) {
 }
 
 func Index(w http.ResponseWriter, r *http.Request) {
+
+	n := Funcionario{}
```

Comment:
this variable 'n' doesn't tells me anything, please use a name that makes sense

Reply:
I removed it